src/securityPolicy.c

```diff
@@ -103,6 +103,7 @@ security_policy_t policyForGetExtendedPublicKey(const bip44_path_t* pathSpec)
 		break;
 
 	default:
+		// TODO should multisig keys be allowed here?
```

Comment:
I think we should treat multisig keys in the same way as ordinary keys here (the potential differences are hidden in `bip44_isPathReasonable`).